src/main/java/Kowalski.java

```diff
@@ -0,0 +1,76 @@
+import java.util.Scanner;
+
+public class Kowalski {
+    public static void main(String[] args){
+        Task[] currentTasks = new Task[100];
+        String dividingLine = "____________________________________________________________";
+        String userInput;
+        int counter = 0;
```

Comment:
I think 'counter' can be replaced with 'taskCounter', so the name explains the variable meaning.